docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -3746,37 +3746,23 @@ The default is `false`.
     ```
```

Comment:
since we're recommending X_SNAP but the default is FAST, should the examples ^ be changed to use X_SNAP?